src/sentry/api/endpoints/relay/register_challenge.py

```diff
@@ -22,6 +23,15 @@ class RelayRegisterChallengeEndpoint(Endpoint):
     authentication_classes = ()
     permission_classes = ()
 
+    enforce_rate_limit = True
+    rate_limits = {
+        "default": {
+            RateLimitCategory.IP: RateLimit(200, 1),
```

Comment:
_nit_: Add a constant somewhere in `endpoints/relay/` to indicate which limits belong together.